acceptance-tests/src/test/java/hudson/plugins/gradle/AbstractAcceptanceTest.java

```diff
@@ -70,6 +70,7 @@ protected final void enableBuildScansForGradle(URI server, String agentVersion)
             settings.clickBuildScansInjection();
             settings.setGradleEnterpriseServerUrl(server);
             settings.setGradleEnterprisePluginVersion(agentVersion);
+            settings.setGradleEnterpriseAccessKey(String.format("%s=secret", server.getHost()));
```

Comment:
Setting `accessKey` triggers the execution of code paths that were not executed before. These code paths were referring to the unavailable class.